packages/suite/src/components/wallet/InputError.tsx

```diff
@@ -11,7 +12,7 @@ const Wrapper = styled.div`
 `;
 
 interface InputErrorProps {
-    button?: { onClick: MouseEventHandler<HTMLButtonElement>; text: string } | { url: string };
+    button?: { onClick: MouseEventHandler<HTMLButtonElement>; text: string } | { url: Url };
```

Comment:
This is a bit funny. 

Why not just:

```suggestion
    button?: { onClick: MouseEventHandler<HTMLButtonElement>; text: string,  url: Url };
```

But its a nit.

Reply:
It is intended because you either want a URL or a click handler, never both at the same time. I made this more apparent in ea22b6a95f4d1a2f63cc7e0f44918313274bb183.

Reply:
Sorry my bad, I had `&` in my mind instead of `|` for some reason. This makes sense, :+1: